### PR TITLE
bugfix: when pr exists, get the PR

### PR DESCRIPTION
**Pull request by dw511214992**

### CODE REVIEW

**Review by zzvswxy — APPROVED** [review]
looks good for me, and pipeline runs good with this change

File: tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml [hunk 1]
```diff
@@ -96,6 +96,18 @@ steps:
 
           prNumber=`echo ${RESULT} | jq -r '.number'`
           prLink=`echo ${RESULT} | jq -r '.html_url'`
+          if [ 10 -gt ${#prLink} ];then
```
Comment (weshaggard) [inline tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml:99]: @zzvswxy @dw511214992 I know I mentioned https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Submit-PullRequest.ps1 in another PR did you ever file a tracking issue for using that over creating another custom implementation here?

Comment (zzvswxy) [reply tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml:99]: Yes, there is a task in Renhe's team github backlog to track this issue:
image
I will create a issue to associate it.
Sorry for the delay,  preparing the inner bugbash takes a lot of Wei Dong and my time.

Comment (dw511214992) [reply tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml:99]: @weshaggard Zhou is working on replacing it with `create-pull-request.yaml` , and this PR is only a hotfix and makes sure the whole workflow can work as expected. Thanks

Comment (weshaggard) [reply tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml:99]: Thanks for the information I mostly wanted to be sure it was tracked. Yes calling Submit-PullRequest.ps1 directly give a little more flexibility and will be fine for this context.

**Pull request merged**